src/Compilers/Core/CodeAnalysisTest/EditorConfigTests.cs

```diff
@@ -0,0 +1,217 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using Roslyn.Test.Utilities;
+using Roslyn.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.UnitTests
+{
+    public class EditorConfigTests
+    {
+        private static EditorConfig ParseConfigFile(string text) => EditorConfig.Parse(text, "");
```

Comment:
>private static EditorConfig ParseConfigFile(string text) => EditorConfig.Parse(text, ""); [](start = 8, length = 89)

Should this use the ` {get; } = ` style to avoid re-parsing an empty string on every call?